delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/model/FileRejection.scala

```diff
@@ -153,6 +153,13 @@ object FileRejection {
   final case class InvalidMultipartFieldName(id: Iri)
       extends FileRejection(s"File '$id' payload a Multipart/Form-Data without a 'file' part.")
 
+  /**
+    * Rejection returned when attempting to create/update a file with a Multipart/Form-Data payload that has user
+    * metadata which cannot be parsed
+    */
+  final case class InvalidUserMetadata(err: String)
```

Comment:
InvalidFileDescription ?
As it has been renamed ?

Reply:
I'll call it `InvalidKeywords` for now, but might re-use it for the name/description later so it might get renamed back